src/main/java/net/glowstone/chunk/GlowChunk.java

```diff
@@ -170,7 +177,17 @@ public GlowBlock getBlock(int x, int y, int z) {
 
     @Override
     public boolean isSlimeChunk() {
-        return false; // TODO: implement slime chunks
+        if (isSlimeChunk == -1) {
+            boolean isSlimeChunk = new Random(this.world.getSeed()
+                    + (long) (this.x * this.x * 0x4c1906)
+                    + (long) (this.x * 0x5ac0db)
+                    + (long) (this.z * this.z) * 0x4307a7L
+                    + (long) (this.z * 0x5f24f) ^ 0x3ad8025f).nextInt(10) == 0;
+
```

Comment:
Could you please inline java.util.Random's formula? This method may end up being called more frequently than we can afford to create objects!

Reply:
Thats why i added the cache :) or what do you mean?

Reply:
What I mean is to calculate what a new `Random` created with the given seed *would* output on the first call to `nextInt(10)`, without actually calling the constructor of `Random`.

Reply:
On second thought, if you can't do this, it's okay; we'll optimize it later.

Reply:
Maybe give the Object in Constructor? That way you only clone it and set the seed per chunk.

Reply:
My concern was about the allocation and reclamation of memory, which would happen just the same whether the object's constructor ran or not.

Reply:
Ah ok. Now its saved in the Regionfiles